doc/getting_started/getting_started.rst

```diff
@@ -35,6 +35,7 @@ You will need to have a `colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-T
   cd ~/ws_moveit2/src
 
   git clone https://github.com/ros-planning/moveit2.git
+  git clone https://github.com/ros-planning/moveit2_tutorials.git
   wget https://raw.githubusercontent.com/ros-planning/moveit2/main/moveit2.repos
   vcs import < moveit2.repos
   wget https://raw.githubusercontent.com/ros-planning/moveit2_tutorials/main/moveit2_tutorials.repos
```

Comment:
```suggestion
  git clone https://github.com/ros-planning/moveit2.git
  git clone https://github.com/ros-planning/moveit2_tutorials.git
  vcs import < moveit2/moveit2.repos
  vcs import < moveit2_tutorials/moveit2_tutorials.repos
```

Reply:
Also do you mind removing `cd ~/ws_moveit2/src` in line 48. That is also redundant.

Reply:
Done!